ocs_ci/ocs/ui/pvc_ui.py

```diff
@@ -222,7 +270,19 @@ def delete_pvc_ui(self, pvc_name, project_name):
         logger.info(f"Select test project {project_name}")
         self.do_click(self.pvc_loc["pvc_project_selector"])
         self.do_send_keys(self.pvc_loc["search-project"], text=project_name)
-        self.do_click(format_locator(self.pvc_loc["test-project-link"], project_name))
+        wait_for_project = self.wait_until_expected_text_is_found(
+            locator=format_locator(self.pvc_loc["test-project-link"], project_name),
+            expected_text=f"{project_name}",
+            timeout=5,
+        )
+        if wait_for_project:
+            logger.info(f"Select test project {project_name}")
+            self.do_click(
+                format_locator(self.pvc_loc["test-project-link"], project_name)
+            )
+        else:
+            logger.error(f"Project {project_name} not found on UI")
+            raise NoSuchElementException
```

Comment:
Can you add a log to the exception? like `raise NoSuchElementException(f"Project {project_name} not found on UI")`